ppss.yaml

```diff
@@ -10,7 +10,7 @@ lake_ss:
     - binance BTC/USDT ETH/USDT 5m
   #    - binance BTC/USDT ETH/USDT BNB/USDT XRP/USDT ADA/USDT DOGE/USDT SOL/USDT LTC/USDT TRX/USDT DOT/USDT 5m
   #    - kraken BTC/USDT 5m
-  st_timestr: 30 days ago # starting date for data
```

Comment:
is this intentional or was it supposed to happen in my_ppss.yaml instead?

Reply:
Mistake, nice catch!

Resolved.